packages/block-library/src/columns/edit.js

```diff
@@ -43,6 +46,65 @@ import {
 */
 const ALLOWED_BLOCKS = [ 'core/column' ];
 
+/**
+ * Template option choices for predefined columns layouts.
+ *
+ * @constant
+ * @type {Array}
+ */
+const TEMPLATE_OPTIONS = [
```

Comment:
Not for now, but this is something that should be extendable like `registerTemplate( blockName, template )`. Ideally it's modelled with full page templates in mind (either `registerBlockTemplate` and `registerTemplate` or assuming a post context instead of block name).